app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False):
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+
+    def hide(self):
+        if self.hidden is False:
+            self.hidden = not False
+        elif self.hidden is True:
+            self.hidden = not True
+        return self
```

Comment:
you can write this in one line

Reply:
I changed this func to this:
def hide(self):
        self.hidden = True if self.hidden is False else False
        
and now I have a proble with pytest, but I don't understand why?(

Reply:
Try to write like this
` self.hidden = not  self.hidden`

Reply:
![image](https://user-images.githubusercontent.com/108726137/183666488-b7af4e58-94c4-4c4c-a83f-42c11ffa362c.png)
why am I getting these?

Reply:
push this code please